Resources/Changelog/GabyChangelog.yml

```
@@ -1,77 +1,238 @@
+Name: Gaby
+Order: -2
 Entries:
 - author: Dreykor
   time: '2024-11-6T12:01:00.0000000+00:00'
-  url: https://github.com/Scomossor/Gaby-Station/pull/2 #url se tiver
+  url: https://github.com/Scomossor/Gaby-Station/pull/2
   changes:
-    - type: Add # One of the following: Add, Remove, Tweak, Fix
+    - type: Add
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)